infrastructure/ansible/roles/ats_sim/tasks/ats_sim.yml

```diff
@@ -0,0 +1,66 @@
+---
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+- name: Install Node.js
+  yum:
+    name: "nodejs"
+    state: installed
+    update_cache : yes
+    lock_timeout: 300
+    enablerepo: "{{ additional_yum_repos | default([]) }}"
+
+- name: Create install directory
+  file:
+    path: "{{sim_install_dir}}"
+    state: directory
+    owner: root
+    group: root
+
+- name: Copy environment file
+  template:
+    src: "env.j2"
+    dest: "{{ sim_install_dir }}/env"
+    owner: root
+    group: root
+    mode: 0644
+
+- name: Copy unit file
+  template:
+    src: "simulator.service.j2"
+    dest: "/lib/systemd/system/simulator.service"
+    owner: root
+    group: root
+    mode: 0555
+
+- name: Git ats_sim.js file
+  git:
+    repo: "{{ sim_git_repo }}"
+    dest: /tmp/tc
```

Comment:
There is a specific module to generate a temp directory, rather than hard coding one.

Reply:
done